docs/samples/v1beta1/transformer/torchserve_image_transformer/README.md

```diff
@@ -142,3 +142,31 @@ Handling connection for 8080
 * Connection #0 to host localhost left intact
 {"predictions": [2]}
 ```
+
+
+
+## Testing with a local transformer & predictor in Kubernetes (developer guide)
+If you are making changes to the code and would like to test running a transformer on your local machine with the predictor running in Kubernetes,
+
+(1) Follow the above steps to deploy the transformer.yaml to Kubernetes. You can comment out the transformer section in the YAML if you wish to only deploy the predictor to Kubernetes.
+
+(2) Run `pip install -e .` in this directory. If you are using a local version of Kserve, you should also run this command in the appropriate directory with a setup.py file so that you pick up any changes from there. 
+
+(3) Port-forward the predictor pod by running 
+```
+kubectl port-forward pods/{name-of-transformer-pod} 8081:8080 
+```
+Since the predictor pod will expose 8080, pick another port to use with localhost. Here, 8081 is used.
```

Comment:
```suggestion
Since the transformer pod will expose 8080, pick another port to use with localhost. Here, 8081 is used.
```

Reply:
here I meant transformer. if you hit localhost:8080. so the predictor needs to expose another port on localhost.